test/memcache-helper.js

```diff
@@ -19,7 +19,14 @@ var config = {
     maxBadLogins: 2,
     maxBadLoginsPerIp: Number(process.env.MAX_BAD_LOGINS_PER_IP) || 3,
     ipRateLimitIntervalSeconds: Number(process.env.IP_RATE_LIMIT_INTERVAL_SECONDS) || 60 * 15,
-    ipRateLimitBanDurationSeconds: Number(process.env.IP_RATE_LIMIT_BAN_DURATION_SECONDS) || 60 * 15
+    ipRateLimitBanDurationSeconds: Number(process.env.IP_RATE_LIMIT_BAN_DURATION_SECONDS) || 60 * 15,
+    badLoginLockout: 3,
+    badLoginLockoutIntervalSeconds: 20,
+    uidRateLimit: {
+      limitIntervalSeconds: Number(process.env.UID_RATE_LIMIT_INTERVAL_SECONDS) || 60 * 15,
+      banDurationSeconds: Number(process.env.UID_RATE_LIMIT_BAN_DURATION_SECONDS) || 60 * 15,
+      maxChecks: Number(process.env.UID_RATE_LIMIT) || 3
+    }
```

Comment:
> are all in flat top-level properties

Yeah it's different but I'm hoping we can nest future once like we do `uidRateLimit`